cyences_app_for_splunk/appserver/static/cs_forensics.js

```diff
@@ -10,51 +10,51 @@ require([
 
     let all_alerts = {
         "Ransomware - Spike in File Writes": {
-            contributing_events: 'index=* tag=endpoint tag=filesystem action=created',
+            contributing_events: '| datamodel Endpoint Filesystem search strict_fields=false | search Filesystem.action=created | `drop_dm_object_name(Filesystem)`',
             system_compromised_search: "| stats sum(count) as count by dest",
-            system_compromised_drilldown: 'index=* tag=endpoint tag=filesystem action=created dest=$row.dest$'
+            system_compromised_drilldown: '| datamodel Endpoint Filesystem search strict_fields=false | search Filesystem.action=created Filesystem.dest=$row.dest$ | `drop_dm_object_name(Filesystem)`'
         },
         "Ransomware - Endpoint Compromise - Fake Windows Processes": {
-            contributing_events: 'index=* tag=process tag=report  process_path!="C:\\\\Windows\\\\System32*" process_path!="C:\\\\Windows\\\\SysWOW64*" | lookup update=true is_windows_system_file filename as process_name OUTPUT systemFile | search systemFile=true',
+            contributing_events: '| datamodel Endpoint Processes search strict_fields=false | search Processes.process_path!="C:\\\\Windows\\\\System32*" Processes.process_path!="C:\\\\Windows\\\\SysWOW64*" | `drop_dm_object_name(Processes)` | lookup update=true is_windows_system_file filename as process_name OUTPUT systemFile | search systemFile=true',
```

Comment:
We are getting different results for both the queries.

Results from OLD Query:
[http://10.33.1.41:8000/en-US/app/search/search?earliest=-15m&latest=now&q=%7C%20datamodel%20Endpoint%20Processes%20search%20strict_fields%3Dfalse%20%7C%20search%20Processes.process_path!%3D%22C%3A%5C%5C%5C%5CWindows%5C%5C%5C%5CSystem32*%22%20Processes.process_path!%3D%22C%3A%5C%5C%5C%5CWindows%5C%5C%5C%5CSysWOW64*%22%20%7C%20%60drop_dm_object_name(Processes)%60%20%7C%20lookup%20update%3Dtrue%20is_windows_system_file%20filename%20as%20process_name%20OUTPUT%20systemFile%20%7C%20search%20systemFile%3Dtrue%20%7C%20stats%20count%20by%20process_name&display.page.search.mode=smart&dispatch.sample_ratio=1&workload_pool=&display.statistics.sortColumn=count&display.statistics.sortDirection=desc&sid=1625564718.13865&display.page.search.tab=statistics&display.general.type=statistics](url)

Results from new Query:
[http://10.33.1.41:8000/en-US/app/search/search?earliest=-15m&latest=now&q=%7C%20datamodel%20Endpoint%20Processes%20search%20strict_fields%3Dfalse%20%7C%20search%20Processes.process_path!%3D%22C%3A%5C%5C%5C%5CWindows%5C%5C%5C%5CSystem32*%22%20Processes.process_path!%3D%22C%3A%5C%5C%5C%5CWindows%5C%5C%5C%5CSysWOW64*%22%20%7C%20%60drop_dm_object_name(Processes)%60%20%7C%20lookup%20update%3Dtrue%20is_windows_system_file%20filename%20as%20process_name%20OUTPUT%20systemFile%20%7C%20search%20systemFile%3Dtrue%20%7C%20stats%20count%20by%20process_name&display.page.search.mode=smart&dispatch.sample_ratio=1&workload_pool=&display.statistics.sortColumn=count&display.statistics.sortDirection=desc&sid=1625564718.13865&display.page.search.tab=statistics&display.general.type=statistics](url)